codegen/templates/tchannel_client.tmpl

```diff
@@ -150,6 +147,7 @@ type {{$clientName}} struct {
 			}
 		}
 		if err != nil {
+			logger.Warn("{{$clientName}}.{{$methodName}} ({{$serviceMethod}}) returned error", zap.Error(err))
```

Comment:
we do not put meta data in the log message. all of these should be fields the log string needs to be static.

Reply:
Actually, they might be already there, since the logger scoped the method.

Reply:
if you want to drop them then that's fine too. We still want a static string non-dynamic here.

Reply:
Done.